docs/tutorials/randomized_benchmarking.rst

```diff
@@ -304,6 +309,211 @@ Running a 2-qubit interleaved RB experiment
+    qc2.draw()
+
+.. jupyter-execute::
+
+    lengths = np.arange(2, 810, 200)
+    num_samples = 30
+    seed = 1011
+    qubits = (0,1)
+
+    # Run a MRB experiment on qubits 0, 1
+    exp_2q = MirrorRB(qubits, lengths, backend=backend, num_samples=num_samples, seed=seed)
+    expdata_2q = exp_2q.run(backend).block_for_results()
+    results_2q = expdata_2q.analysis_results()
+
+.. jupyter-execute::
+
+    # View result data
+    print("Gate error ratio: %s" % expdata_2q.experiment.analysis.options.gate_error_ratio)
+    display(expdata_2q.figure(0))
+    for result in results_2q:
+        print(result)
+
+
+Selecting :math:`y`-axis values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. jupyter-execute::
+
+    lengths = [2, 52, 102, 152]
+    num_samples = 30
+    seed = 42
+    qubits = (0,)
+
+    exp = MirrorRB(qubits, lengths, backend=backend, num_samples=num_samples, seed=seed)
+    # select y-axis
+    exp.analysis.set_options(y_axis="Success Probability") # or "Adjusted Success Probability" or "Effective Polarization"
+    # y-axis label must be set separately
+    exp.analysis.options.curve_drawer.set_options(
+    #     xlabel="Clifford Length",
+        ylabel="Success Probability",
+    )
+    expdata = exp.run(backend).block_for_results()
+    results = expdata.analysis_results()
+
+.. jupyter-execute::
+
+    display(expdata.figure(0))
+    for result in results:
+        print(result)
+
+
+Mirror RB user options
+~~~~~~~~~~~~~~~~~~~~~~
+
+Circuit generation options can be specified when a ``MirrorRB`` experiment 
+object is instantiated: 
+
+- ``local_clifford`` (default ``True``): if ``True``, begin the circuit with 
+  uniformly random one-qubit Cliffords and end the circuit with their inverses
+
+- ``pauli_randomize`` (default ``True``): if ``True``, put layers of uniformly 
+  random Paulis between the intermediate Clifford layers
+
+- ``two_qubit_gate_density`` (default ``0.2``): expected fraction of two-qubit 
+  gates in each intermediate Clifford layer
+
+- ``inverting_pauli_layer`` (default ``False``): if ``True``, put a layer of 
+  Paulis at the end of the circuit to set the output to 
+  :math:`\left\vert0\right\rangle^{\otimes n}`, up to a global phase
+
+
+Mirror RB implementation in ``pyGSTi``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``MirrorRBPyGSTi`` subclass of ``MirrorRB`` uses the circuit generation in 
+``pyGSTi`` but the circuit transpilation in Qiskit Experiments. It is primarily 
+used for testing and comparison, and an instance of such an experiment is 
+constructed in the same way as described above. ``MirrorRBPyGSTi`` uses Qiskit's 
+transpilation because ``pyGSTi`` transpiles circuits slightly differently, 
+producing small discrepancies in fit parameters between the two codes. To 
+illustrate, consider the two circuits below, both of which were generated in 
+``pyGSTi``. The first circuit was transpiled in ``pyGSTi``, 
+
+.. image:: pygsti-data-pygsti-transpiled-circ.png
```

Comment:
Why this cannot be generated here? You have hidden cell option.

Reply:
pyGSTi is required to generate these circuits, and we don't want to require pyGSTi to accurately render the documentation.